homeassistant/components/light/__init__.py

```diff
@@ -198,6 +220,8 @@ def handle_light_service(service):
             service_fun = 'turn_off'
         elif service.service == SERVICE_TOGGLE:
             service_fun = 'toggle'
+        elif service.service == SERVICE_SET_BRIGHTNESS:
+            service_fun = 'set_brightness'
```

Comment:
The Light ABC is the Light class starting below this method. It should have a `set_brightness` method that raises a not implemented exception. [Like this](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/helpers/entity.py#L234-L236)